### PR TITLE
Update blog post limit

### LINKED ISSUE #3676: blog tab not working after page 5
I was exploring some older posts on the blog (www.publiclab.org/blog) when I noticed I couldn't tab back more than the 4 pages that show on the bottom of the blog. If you try to hit "next" once you're on page 4 nothing happens. 

screen shot 2018-10-11 at 4 31 51 pm

@publiclab/mentors here's another bug that might be made into a good issue :)

### PR DESCRIPTION
**Pull request by shubhangi-chauhan**
Increased the limit on blog posts, set it to number of notes with tag 'blog'.

Fixes #3676 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by shubhangi-chauhan** [comment]
@publiclab/reviewers @jywarren 
Please review.

File: app/controllers/tag_controller.rb [hunk 1]
```diff
@@ -246,7 +246,8 @@ def widget
   end
 
   def blog
-    nids = Tag.find_nodes_by_type(params[:id], 'note', 20).collect(&:nid)
+    blog_count = Tag.tagged_node_count('blog')
```
Comment (jywarren) [inline app/controllers/tag_controller.rb]: So sorry I missed this! I think we can actually skip this line and specify a length of `nil` -- that'll remove the limit!

**Comment by SidharthBansal** [comment]
@shubhangi-chauhan do you need any help on this issue?

**Comment by SidharthBansal** [comment]
As the person is inactive for more than a month, I am closing the PR. In case you want to push changes please feel free to open a new PR OR reopen this PR and add additional changes to it.
Thanks for contributing on Public Lab

**Comment by gauravano** [comment]
I will complete this PR @SidharthBansal . Thanks!

**Comment by SidharthBansal** [comment]
Great

**Comment by gauravano** [comment]
@jywarren also tested on unstable.publiclab.org too

Peek 2019-03-22 22-15

**Comment by jywarren** [comment]
Super, thank you!!!!

**Pull request merged**